documentation/docs/release-notes/3.1.0.md

```diff
@@ -17,19 +17,39 @@ PMM empowers you to:
 
 ## Release highlights
 
-### Improved MongoDB cluster topology visualization
+### Improvements to MongoDB monitoring
 
-We have improved the [MongoDB Cluster Summary dashboard](../reference/dashboards/dashboard-mongodb-cluster-summary.md) by restoring and refining the **Current Topology** section, which had been available in older PMM 2 versions.
+#### Improved MongoDB cluster topology visualization
 
-This section now correctly organizes and displays MongoDB cluster components—including shards, config servers, and mongos routers—within their respective sections, ensuring a clear hierarchical structure. Previously, all components were incorrectly grouped under the Config Services section.
+We have improved the [MongoDB Cluster Summary dashboard](../reference/dashboards/dashboard-mongodb-cluster-summary.md) with an enhanced **Current Topology** section, which had been available in older PMM 2 versions.
 
-This enhancement provides a more accurate and intuitive visualization of your MongoDB cluster architecture, making it easier to understand, monitor, and manage the relationships between its components:
+This redesigned section now properly categorizes MongoDB cluster components—shards, config servers, and mongos routers—within their appropriate hierarchical sections. This replaces the previous implementation where all components were incorrectly grouped under the Config Services section.
```

Comment:
Not sure, but maybe it should be "config servers"?

Reply:
yes, seems like this is Config Servers indeed
<img width="1599" alt="image" src="https://github.com/user-attachments/assets/ce24ea88-88db-495a-bedf-d3fc76ff8eab" />